zend_abstract_interface/uri_normalization/uri_normalization.c

```diff
@@ -215,6 +215,7 @@ bool zai_match_regex(zend_string *pattern, zend_string *subject) {
     zai_error_state error_state;
     zai_sandbox_error_state_backup(&error_state);
     zend_replace_error_handling(EH_NORMAL, NULL, NULL);
+    EG(error_reporting) = 0;
```

Comment:
Why are we adding the `EG(error_reporting) = 0;` instruction here? Isn't `zend_replace_error_handling(EH_NORMAL, NULL, NULL);` already doing that considering that `EH_NORMAL` equals zero

Reply:
Oh yeah, misread `error_handling` for `error_reporting` 😬